wandb/util.py

```diff
@@ -1318,7 +1318,7 @@ def is_unicode_safe(stream):
     """returns true if the stream supports UTF-8"""
     if not hasattr(stream, "encoding"):
         return False
-    return stream.encoding == "UTF-8"
```

Comment:
if you already fixing this function would you mind also adding this function here:
https://github.com/wandb/client/blob/master/wandb/sdk/wandb_run.py#L2144
for consistency?

Reply:
but then i cant call it a oneliner :)
done.
There are more cases in the code, but im leaving it for a future refactor.

Reply:
yeah this one came to top my mind since i saw it earlier today